meshroom/nodes/aliceVision/KeyframeSelection.py

```diff
@@ -65,84 +65,12 @@ class KeyframeSelection(desc.CommandLineNode):
             label="mmFocals",
             description="Focals in mm (will be use if not 0)."
         ),
-        desc.ListAttribute(
```

Comment:
We need to generate images with the right metadata.

Reply:
I do exactly what was done before ? Except we don't use anymore focal in pixels.